packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -9,31 +9,37 @@ import defaultClasses from './miniCart.css';
 
 import { mergeClasses } from '../../classify';
 import getCurrencyCode from '../../util/getCurrencyCode';
+import { useAppContext } from '@magento/peregrine/lib/context/app';
+import { useCartContext } from '@magento/peregrine/lib/context/cart';
+import { useCheckoutContext } from '@magento/peregrine/lib/context/checkout';
 
 const MiniCart = props => {
+    const [, { closeDrawer }] = useAppContext();
+    const [
+        cartState,
```

Comment:
Thanks for calling this `cartState` rather than `cart`, and updating the references throughout. I know that's minor, but I think it will help people trying to follow what's going on in these components.